dashboard/src/modules/components/OverviewComponent/common-component.jsx

```diff
@@ -99,7 +99,7 @@ export const NewRunsHeading = () => {
   };
   return (
     <div className="newruns-heading-container">
-      <Heading title="New and unmanaged runs" />
+      {/* <Heading title="New and unmanaged runs" /> */}
```

Comment:
It doesn't seem useful to "render" this comment. If you're not using the header, it'd be better to delete this line.

Reply:
Removed it